Tools/Schema.NET.Tool/Schema.NET.Tool.csproj

```diff
@@ -3,12 +3,33 @@
   <PropertyGroup Label="Build">
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>false</IsPackable>
+    <IsRoslynComponent>true</IsRoslynComponent>
   </PropertyGroup>
 
   <ItemGroup Label="Package References">
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.10.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(BuildingInsideVisualStudio)' == 'true'">
```

Comment:
This looks pretty hacky. Worth adding a comment as to why we had to do this? Link to a GitHub issue maybe?

Reply:
Good idea about adding a comment - agreed the solution is pretty hacky. It might be a race to whether the Visual Studio/MSBuild handling "fixes" the issue or the .NET SDK does.